### ISSUE TITLE
Create content for "Special Report for Forbes Asia"

### ISSUE THREAD

**Issue by lapin7** [rchain/bounties#581]
RChain received this request from Global Group Media for Forbes Asia. Let's prepare an info package about RChain. Something like [this](https://drive.google.com/open?id=1odSk4SvCPgr-L9MsTyKFFdTDVeRW5049)

- [ ] Find out what the conditions are: deadline, $fees
- [ ] Add links to info that you think is useful
- [ ] Create a Gdoc summary Pitch about RChain
- [ ] Provide links to other target magazines
- [ ] Make other SMART issues that relate to this issue
- [ ] Suggested Budget: $2000
- [ ] Duration 2 weeks
- [ ] Suggested Participants: @rchain/marketing-pr @patrick727 @kitblake @dckc @llerner @Mervyn853 @pmoorman @AyAyRon-P @mrinalmanohar ..... or write a comment to be added

> To Whom It May Concern,

> My name is Sophie Lin, Project Director for Global Group Media providing content for Forbes Asia. I am contacting you today to invite you to participate in a Special Report for Forbes Asia. As innovators in economic reporting, Forbes has a long heritage of bringing the latest financial investment information to a specified audience.

> After researching RChain, it has come to my attention that your brand’s message may be a great fit for promotion in Forbes. In this Special Report you are able to provide all and any information that you wish to provide, you have carte blanche. The thriving Asian market is looking to invest in the latest in cryptocurrencies. The Japanese market alone is #1 in cryptocurrency trading according to a report by the Wall Street Journal in November 2017. One third of working South Koreans have invested in cryptocurrency according to Thomson Reuters. Emerging markets such as India, Malaysia and Indonesia are rapidly following in Japan and South Korea’s footsteps.

> Global Group Media offers both online and in print presence in our pricing. Also included in our services is a team of professionals who can write, design and format your report for print and online.  I have attached our deck for your review.

> By participating in this Special Report you are streamlining your marketing towards a specified audience and market. This is not a “shot in the dark” campaign. With a name like Forbes behind you, your brand message will be heard loud and clear by those you want to speak to.

> I thank your in advance for your careful consideration and please feel free to contact me at any time to further discuss any details.

 

> Best Regards,

> Sophie Lin
Project Director

> Global Group Media
London
WhatsApp: +1 (312) 619-0577
Website: www.GlobalGroupMedia.com

**Comment by David405** [comment]
I think a Microsoft PowerPoint presentation would do this perfectly, what do you think @laplin?

**Comment by David405** [comment]
@lapin I would love to be added to the group working on this.

**Comment by David405** [comment]
@lapin7 I think google slides or microsoft powerpoint will do a better job for creating a pitch summary than using google doc.

**Comment by ResonanceXX** [comment]
Wow! This is some amazing news. 
We should play on our diversity. Forbes Asia, is going to be read by a lot of Asians. 
And the coop enjoys a healthy number of contributions from that region. It would also be interesting to point out the many other benefits of RChain as a global entity.
**Diversity** and our innovative approach to Blockchain development. Ease of access and the core principle of decentralization within the coop too. China, especially, has a lot of censorship and it would be nice to see plenty of the more than 1 billion citizens there hop on this wonderful train.

**Comment by David405** [comment]
Nice point @ResonanceXX. Africa also has a ready market for RChain and so we should also work towards marketing Africa. I believe we could equally get featured on <href a="https://www.forbesafrica.com/">Forbes Africa</a>.

**Comment by David405** [comment]
@lapin7 I believe the fees were declared in page 37 of the pdf you gave.

**Comment by ResonanceXX** [comment]
With plenty of optimism. I believe in that being possible. 

Sent from Yahoo Mail on Android 
 
  On Mon, 9 Apr 2018 at 15:26, David405<notifications@github.com> wrote:   
Nice point @ResonanceXX. Africa also has a ready market for RChain and so we should also work towards marketing Africa. I believe we could equally get featured on Forbes Africa.

**Comment by lapin7** [comment]
> @David405 I believe the fees were declared in page 37 of the pdf you gave

You think it's this?

# GGM Sponsorship Packages for Forbes Asia

SECTOR EXCLUSIVE Double Page USD 55,000
PLATINUM One Full Page USD 35,000
GOLD Half (1⁄2) Page USD 25,000
SILVER Third (1⁄3) Page USD 17,000
BRONZE Quarter (1⁄4) Page USD 15,000
BANNER MENTION Online Only* USD 10,000
LOGO FEATURE Online Only * USD 7,500

**Comment by David405** [comment]
Yes I believe @lapin7, in the mail written by the project director, this was mentioned where she spoke about online and print pricing which describes those fees.

**Comment by pmoorman** [comment]
Guys, I think this Forbes just selling advertising packages. While there is nothing intrinsically wrong with that, I think our first question should be:

**Is it a good ROI to advertise RChain on Forbes Asia?**

Does it look like this kind of campaign will yield a return?
I'm fairly skeptical that we'll make back $20-50k+ in display advertising like this. We're a startup and not an established brand. We need different sorts of marketing efforts.

_**TL;DR:** I think pursuing this is wasted money & effort._

**Comment by lapin7** [comment]
@pmoorman is right. The ROI of advertising RChain on Forbes Asia is probably close to 0.

**Let's close this issue** and move on with our own marketing efforts.
